Jenkinsfile

```diff
@@ -23,21 +23,43 @@ pipeline {
         }  
         
         
-        stage('Publish Docker image') {
+        stage('Publish Docker images') {
+
+            // This when clause disables PR build uploads; you may comment this out if you want your build uploaded.
+            when {
+                beforeAgent true
+                not {
+                    changeRequest()
+                }
+            }
 
             steps {
                 withDockerRegistry([url: 'https://index.docker.io/v1/', credentialsId: 'docker.com-bot']) {
                     sh '''#!/usr/bin/env bash
-                        if [ -z $CHANGE_ID ]; then
-                            echo "Publishing docker images for Eclipse Codewind ..."
-                            ./scripts/publish.sh imageName registry tag
+                        echo "Publishing docker images for Eclipse Codewind ..."
+                        export REGISTRY="eclipse"
+                        echo "Branch name is $GIT_BRANCH"
+
+                        if [ "$GIT_BRANCH"="master" ]; then
```

Comment:
You have to put spaces around the `=`.

Reply:
I would write it as `if [[ $GIT_BRANCH == "master" ]];`, double `[` is more predictable and you don't have to quote variables

Reply:
Fixed as you wish :)